CRM/Utils/Check/Component/Security.php

```diff
@@ -213,8 +213,8 @@ public function checkSigningKey(): array {
         ts('Some components and extensions may need to generate cryptographic signatures. Please configure <a %1>CIVICRM_SIGN_KEYS</a>. ',
           [1 => 'href="https://docs.civicrm.org/sysadmin/en/latest/setup/secret-keys/" target="_blank"']
         ),
-        ts('Signing Key Recommended'),
-        \Psr\Log\LogLevel::NOTICE,
+        ts('Signing Key Strongly Recommended'),
```

Comment:
```suggestion
        ts('Signing Key Missing'),
```

Reply:
Right, "Strongly Recommended" is verbose. "Missing" reads a little weird to me, because none of the other headings end with "-ing", and "Signing Key Missing" has two "-ing"s.

How about "Signing Key Undefined" or just plain "Signing Key" (or go back to just "Signing Key Recommended")?

<img width="947" alt="Screen Shot 2023-01-05 at 4 03 11 PM" src="https://user-images.githubusercontent.com/1336047/210903009-7290dc29-cb94-4e4d-af2e-82e9c090ead0.png">

or

<img width="947" alt="Screen Shot 2023-01-05 at 4 03 40 PM" src="https://user-images.githubusercontent.com/1336047/210903056-6678db90-cdb5-4e0a-8120-76805a79db1c.png">

Reply:
Aside: Trimming those titles ("Database Upgrade ~~Required~~", "Debug Mode ~~Enabled~~", "Signing Key ~~Recommended~~", "PHP") would probably make it cleaner/easier to read/easier to compose. It would be harder to google on on the message-title, but then I think people would google for the text instead...